src/pages/LoginPage/loginPage.css

```diff
@@ -9,6 +9,6 @@
 
 .floatLeft{
     position: absolute;
-    top: 4rem;  
+    top: 1rem;  
     left: 8.2rem; 
 }
```

Comment:
It's a common practice to add a new line at the end of every line. 
WDYT if you do it?